.github/workflows/rust.yml

```diff
@@ -60,11 +60,11 @@ jobs:
         - name: Checkout sources
           uses: actions/checkout@v2.3.3
 
-        - name: Install stable toolchain
+        - name: Install nightly toolchain
```

Comment:
What's the advantage between the stable and the nightly toolchain for clippy?

Reply:
Only that the nightly toolchain has a great deal more lints

Reply:
Actually the more dramatic difference is in rustfmt- nightly fmt is far more powerful